scipy/optimize/_linprog_ip.py

```diff
@@ -724,7 +724,7 @@ def _ip_hsd(A, b, c, c0, alpha0, beta, maxiter, disp, tol, sparse, lstsq,
     if disp:
         _display_iter(rho_p, rho_d, rho_g, "-", rho_mu, obj, header=True)
     if callback is not None:
-        x_o, fun, slack, con, _, _ = _postsolve(x/tau, *_T_o,
+        x_o, fun, slack, con, _, _ = _postsolve(x/tau, postsolve_args,
```

Comment:
In the original, we're unpacking the tuple whereas now we're passing the argument as it's own variable. This shouldn't make any difference given that the order remains the same.